addon/-private/properties/collection.js

```diff
@@ -5,7 +5,7 @@ import { create } from '../create';
 import { count } from './count';
 import Ceibo from 'ceibo';
 
-const arrayDelegateMethods = ['map', 'filter', 'mapBy', 'filterBy', 'forEach'];
+const arrayDelegateMethods = ['map', 'filter', 'find', 'mapBy', 'filterBy', 'findBy', 'forEach'];
```

Comment:
Will this work if the host app has disabled Array prototype extensions?

Reply:
It looks like we're not depending on the host app's prototype extensions and are ensuring that these functions are executed on an Ember Array: https://github.com/san650/ember-cli-page-object/blob/master/addon/-private/properties/collection.js#L23